pkg/framework/host.go

```diff
@@ -496,7 +505,7 @@ func (h *Host) PodName(namespace, labelSelector string) (string, error) {
 	return pod.Name, nil
 }
 
-// RestConfig returns the host cluster framework's rest config.
+// RestConfig returns a deep copy of host cluster framework's rest config.
```

Comment:
<3

Reply:
This is not actually true :) I tried to do the DeepCopy but it returns another type (uses embedded type DeepCopy). Good you commented though so I revert the comment :)